src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -27,6 +27,8 @@ namespace Microsoft.CodeAnalysis.LanguageServer
 {
     internal static class ProtocolConversions
     {
+        // NOTE: While the spec allows it, don't use Function and Method, as both VS and VS Code display them the same way
+        // which can confuse users
```

Comment:
Can we write a test that asserts there's no value in here with kind 'Function'? I could easily imagine the comment being missed if somebody adds something to the end.

Reply:
Good idea